python/lsst/display/firefly/firefly.py

```diff
@@ -125,6 +125,13 @@ def __init__(self, display, verbose=False, url=None,
     def _getRegionLayerId(self):
         return "lsstRegions%s" % self.display.frame if self.display else "None"
 
+    def _clear_image(self):
```

Comment:
I think this should probably be `_clearImage`, since this class inherits from a base class that definitely uses the camelCase conventions used in Science Pipelines code, but it's a bit of a grey area and I won't insist (especially since it's a private method).